src/vaibhav/CoadingExam3/RemoveVowel.java

```diff
@@ -0,0 +1,48 @@
+/*Coding Exam - 2: 26th April 2021
+
+WAP to return a string by removing all the vowels from the given String.
+Output should be printed in main method.
+
+Input: globant india test automation engineer
+
+Output: glbnt nd tst tmtn ngnr
+
+*/
+
+package vaibhav.CoadingExam3;
+
+public class RemoveVowel {
+
+	String processString(String str) {
+		String temp = "";
+		for (int index = 0; index < str.length(); index++) {
+			char ch = str.charAt(index);
+			switch (ch) {
+			case 'a':
```

Comment:
If you use break then it will break the loop, you should write if it's vowel then do not process it else append it to string temp.

https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/1475/files